plugin/trino-hudi/src/test/resources/hudi-testing-data/stock_ticks_cow_clustered/.hoodie/20230712105030944.commit

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Please add README.md `stock_ticks_cow_clustered` directory. The file should contain steps to reproduce for creating the table, e.g. CREATE TABLE and INSERT statement.